apps/cowswap-frontend/src/modules/trade/hooks/useIsQuoteUpdatePossible.ts

```diff
@@ -0,0 +1,22 @@
+import { TradeFormValidation, useGetTradeFormValidation } from 'modules/tradeFormValidation'
+
+import { useIsWrapOrUnwrap } from './useIsWrapOrUnwrap'
+
+const NO_UPDATE_STATES = [
+  TradeFormValidation.CurrencyNotSupported,
+  TradeFormValidation.CurrencyNotSet,
+  TradeFormValidation.InputAmountNotSet,
+]
```

Comment:
Should probably add the one for account not connected if something like this exists.